.travis.yml

```diff
@@ -6,17 +6,22 @@ python:
 addons:
   apt:
     update: true
-env:
-  - PACKAGE_NAME=pytest-confluence-report PACKAGE_VERSION=0.0.2
-before_install:
-  - sudo add-apt-repository -y ppa:duggan/bats
-  - sudo apt-get -y install bats
-install:
-  - pip install pip -U && pip install -r requirements-dev.txt -U
-script:
-  - bats test-package.bats
-  - ./analyse-source-code.sh
-after_success:
-  - coveralls
 notifications:
   email: false
+jobs:
+  include:
+    - stage: build
+      env: PACKAGE_NAME=pytest-confluence-report PACKAGE_VERSION=0.0.2
+      install:
+        - pip install pip -U
+      after_success:
```

Comment:
This part should be included into `unittests` job as we call tests coverage after test execution